l10n_it_codici_carica/models/codice_carica.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
You could use https://github.com/OCA/maintainer-tools/blob/master/template/module/models/__init__.py#L1-L3